transmission/updateSettings.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+import json
+
+
+# Verify script arguments
+if len(sys.argv) != 3:
+    sys.exit('Invalid number of arguments. Usage:\n updateSettings.py defaultSettingsFile.json outputSettingsFile.json')
+
+default_settings = sys.argv[1]
+transmission_settings = sys.argv[2]
+
+if not os.path.isfile(default_settings):
+    sys.exit('Invalid arguments, default settings file does not exist')
+
+# Define which file to base the config on
+if os.path.isfile(transmission_settings):
+    configuration_baseline=transmission_settings
+else:
+    configuration_baseline=default_settings
```

Comment:
Is there a case where you'd like both `default_settings` and `transmission_settings` to be read and then if theres anything in the `transmission_settings` file then those should overwrite the default settings?